src/main.cpp

```diff
@@ -999,11 +999,11 @@ bool CheckSequenceLocks(const CTransaction &tx, int flags, LockPoints* lp, bool
 unsigned int GetLegacySigOpCount(const CTransaction& tx)
 {
     unsigned int nSigOps = 0;
-    BOOST_FOREACH(const CTxIn& txin, tx.vin)
+    for (const CTxIn& txin : tx.vin)
```

Comment:
Why are you using `CTxIn&` here and `CTxOut&` below instead of `auto&` as in other changes? Is there any reason for this I do not see?